### ISSUE TITLE
SVG node failed to parse

### ISSUE THREAD

**Issue by shuding** [vercel/satori#240]
# Bug report

## Description / Observed Behavior

> Had to tweak a few SVG's because otherwise vercel/og wouldn't render them.

```diff
- <svg id="svg" version="1.1" xmlns="http://www.w3.org/2000/svg" width="1.25em" height="1.25em" viewBox="0, 0, 400,400">
+ <svg id="svg" xmlns="http://www.w3.org/2000/svg" width="40" height="40" viewBox="0,0,400,400">
```

https://github.com/calcom/cal.com/pull/4943#discussion_r997985008

**Comment by JeroenReumkens** [comment]
@shuding  I'm still unsure what the reasons were it broke. The exact files I changed are referenced in the PR you mentioned in this issue. So it's not only the SVG you quoted, just so you know 😄 

For most of the SVG's I tweaked a few things at once, and later on I noticed that it was even quicker to use SVGOMG, which fixed it most of the time. I think it was mostly about missing width/height, width/height in something else than px, decimals in width/height (??), something with minification (??), usage of gradients (??). 

The error I got every time was the following: 
![CleanShot 2022-10-19 at 09 35 51@2x](https://user-images.githubusercontent.com/2969573/196626492-2b32d963-36d6-4d00-9efe-b322ebbd968c.jpg)
